src/Plugin/GraphQL/DataProducer/Entity/EntityRendered.php

```diff
@@ -95,19 +95,19 @@ public function __construct(
 
   /**
    * @param \Drupal\Core\Entity\EntityInterface $entity
-   * @param string|null $mode
    * @param \Drupal\Core\Cache\RefinableCacheableDependencyInterface $metadata
+   * @param string|null $mode
    *
    * @return string
    */
-  public function resolve(EntityInterface $entity, $mode = NULL, RefinableCacheableDependencyInterface $metadata) {
+  public function resolve(EntityInterface $entity, RefinableCacheableDependencyInterface $metadata, $mode = NULL) {
```

Comment:
we should not change the parameter order, better just remove the default value of $mode